semgrep_output_v1.atd

```diff
@@ -1680,6 +1686,42 @@ type ci_scan_complete_stats = {
     <python repr="dict">
     <ts repr="map">
     option;
+
+  (* since 1.98.0 *)
+  (* In collaboration with the Data Science team, it was suggested
+   * that we start to group stats by product for organizational purposes.
+   *)
+  ?supply_chain_stats: supply_chain_stats option;
+}
+
+(* Valid schema versions. Update when making breaking changes. *)
+type stats_schema_version <ocaml attr="deriving show"> <python decorator="dataclass(frozen=True)"> = [
+  | V1_0_0 <json name="1.0.0"> (* initial version *)
+] 
+
+type resolution_method <ocaml attr="deriving show"> <python decorator="dataclass(frozen=True, order=True)"> = [
+  | LockfileParsing 
+  | DynamicResolution 
+]
+
+type dependency_resolution_stats = {
+  resolution_method: resolution_method;
+  dependency_count: int;
+  ecosystem: ecosystem;
+}
+
+type subproject_stats = {
+  dependency_source_files: string list;
+  (* Results of dependency resolution, null if resolution failed *)
+  resolved_stats: dependency_resolution_stats nullable;
+}
+
+type supply_chain_stats = {
+  subproject_stats: subproject_stats list;
+  lockfile_scan_info: (string * int) list
```

Comment:
I'm planning to move `lockfile_scan_info` into the supply chain stats object. However considering it's deprecated, does that even make sense? Should I just leave it out?

Reply:
I think it seems duplicative of the info in `subproejct_stats` -- I think I would vote that we just leave it where it is for now, maybe

Reply:
Removed in [0fbb591](https://github.com/semgrep/semgrep-interfaces/pull/313/commits/0fbb591f12b76200f26aa80bc09ea95c43468fc9)